### DISCUSSION TITLE
请求改变TableFilter.cs中的条件数量（_count）访问属性

### DISCUSSION THREAD

**Discussion by ya00ya11** [dotnetcore/BootstrapBlazor#3166] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

在使用Table中的FilterTemplate时发现，自定义的Filter在继承FilterBase后，无法访问TableFilter._count，导致只能使用单条件过滤，同时需要设置 
TableFilter.ShowMoreButton = false
因为在无法访问TableFilter._count后，无法根据 _count 添加/减少过滤条件

### Describe the solution you'd like

将
    /// <summary>
    /// 获得/设置 条件数量
    /// </summary>
    private int _count;
改为
    /// <summary>
    /// 获得/设置 条件数量
    /// </summary>
    public int _count;

**Comment by bb-auto** [comment]
@ya00ya11 Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment, marked as answer]
@ya00ya11 你不需要使用这个啊，你只需要把自己的逻辑写完就行了啊
